resources/PhoneNumberMetadata.xml

```diff
@@ -18777,11 +18777,11 @@
     </territory>
 
     <!-- Lithuania (LT) -->
-    <!-- Note that Lithuania is switching to a national prefix of 0. We support
-         both 0 and 8 when parsing until this switch is complete. -->
+    <!-- Note that Lithuania switched to a national prefix of 0. -->
+    <!-- The former prefix 8, in place since the Soviet times, will continue to be operational until 28 February 2025. -->
```

Comment:
https://en.wikipedia.org/wiki/Telephone_numbers_in_Lithuania#:~:text=Since%201%20March%202024%2C%20Lithuania,uses%20%2B370%20country%20calling%20code.